src/main/java/com/sintern/domain/entity/TestResponse.java

```diff
@@ -0,0 +1,33 @@
+package com.sintern.domain.entity;
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class TestResponse {
+    @Id
+    @GeneratedValue(generator = "uuid2")
+    @GenericGenerator(name = "uuid2", strategy = "uuid2")
+    @Column(columnDefinition = "uuid")
+    UUID id;
+
+    @ManyToOne
+    @JoinColumn(name = "test_id", referencedColumnName = "id")
+    Test test;
+
+    @ManyToOne
+    @JoinColumn(name = "quizz_question_id", referencedColumnName = "id")
+    QuizQuestion quizQuestion;
+
+    String selectedAnswer;
```

Comment:
Change this, please (https://github.com/S-Intern-G4/SIntern/pull/47#discussion_r775220092)